sass/tji/_data.scss

```diff
@@ -0,0 +1,54 @@
+/*--------------------------------------------------------------
+>>> EXPLORE THE DATA TABLE OF CONTENTS:
+----------------------------------------------------------------
+# GENERAL
+# CHARTS
+
+/*--------------------------------------------------------------
+## CHARTS
+--------------------------------------------------------------*/
+
+#js-TJIChartView {
+  width: 100%;
+  padding: 2em;
+  background-color: #fbfbfb;
+  box-shadow: 2px 3px 6px rgba(64,64,64,0.5);
+  margin-left: 1em;
+  
+  h2 {
+    display: block;
+    width: 100%;
+    text-transform: uppercase;
+    padding-left: 0.5em;
+    padding-right: 0.5em;
+    margin-bottom: 0.5em;
+    margin-top: 0;
+  }
+  
+  @media (max-width: 992px) {
+	  margin-left: 0;
+  }
+}
+
+.tji-data-count {
+	font-size: 18px;
+	margin-bottom: 1em;
+}
+
+.tji-chart {
+
+  border: 1px solid #ebebeb;
+  transition: all 0.25s;
+  
+  &:hover {
+    box-shadow: 0px 0px 20px rgba(64,64,64,0.15);
+    background-color: #fff;	    
+  }
+  
+  @media (min-width: 1500px) {
+	  -ms-flex-preferred-size: 33.33%;
```

Comment:
We have flexbox classes for this sort of thing and overwriting library styles likes this is a tricky animal and tends to get confusing if there's a lot of it. So two potential other options so we don't have to go down this path, 
1) we can use https://github.com/hugeinc/flexboxgrid-sass, which is a sass-ified version of flexboxgrid and set our breakpoints so large could be 1500.
2) do we need to worry about people with screens that are larger than 1500? Is that all that common?